pykafka/balancedconsumer.py

```diff
@@ -504,18 +515,24 @@ def _check_held_partitions(self):
             self._rebalance()
 
     def _brokers_changed(self, brokers):
+        if not self._running:
+            return False  # disables the watcher
```

Comment:
If `True` doesn't actually have any meaning to the caller, I think it's actually more readable to leave it out.